### PR TITLE
fix deadlink with the proper one

### PR DESCRIPTION
**Pull request by filimonov**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by filimonov** [comment]
ghhr... still wrong.

**Comment by filimonov** [comment]
It was renamed by 
5250d9ad11a50abdeb0605ce3ba0a29aec3c6f0f
and later removed by
8f01fe9c496386013837da017fb8b8e7dd850651

**Comment by filimonov** [comment]
@rfraposa any thoughts?

**Comment by rfraposa** [comment]
The 2021 changelog.md file is here: https://github.com/ClickHouse/clickhouse-docs/tree/main/docs/en/whats-new/changelog 

FWIW: the first thing I do when I manually deploy the changelog.md file (our CI isn't building the changelog properly for Docusaurus) is delete that TOC at the top of the page, because Docusaurus builds a TOC automatically. So the link you are trying to fix can probably just be deleted anyway.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 **[ClickHouse release v22.3-lts, 2022-03-17](#223)**<br>
 **[ClickHouse release v22.2, 2022-02-17](#222)**<br>
 **[ClickHouse release v22.1, 2022-01-18](#221)**<br>
-**[Changelog for 2021](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/whats-new/changelog/2021.md)**<br>
+**[Changelog for 2021](https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/reference/whats-new/changelog/2021.md)**<br>
```
Comment (DanRoscigno) [inline CHANGELOG.md:7]:
```suggestion
**[Changelog for 2021](https://clickhouse.com/docs/en/whats-new/changelog/2021/)**<br>
```

Comment (DanRoscigno) [reply CHANGELOG.md:7]: When in the docs, we link to the markdown files that are also in the docs, and when Docusaurus generates the HTML it replaces the links to the markdown files with links to the HTML.  It gets easier to grok over time :)  I think the suggestion above will fix it.

Comment (DanRoscigno) [reply CHANGELOG.md:7]: When in the docs, we link to the markdown files that are also in the docs, and when Docusaurus generates the HTML it replaces the links to the markdown files with links to the HTML.  It gets easier to grok over time :)  I think the suggestion above will fix it.

**Review by DanRoscigno — APPROVED** [review]
Link now works.

**Comment by filimonov** [comment]
> The 2021 changelog.md file is here: https://github.com/ClickHouse/clickhouse-docs/tree/main/docs/en/whats-new/changelog 

I still think that having full detailed changelog in the repo itself + in the distro is a nice idea.
Reason - you can analyze it with simple tools like `grep`, `less` etc + available even without the internet connection.

**Comment by filimonov** [comment]
For example https://amoeboids.com/blog/changelog-how-to-write-good-one/ 

> Always include a complete history of notable changes. This lets your changelog serve as a historical reference. Don’t make the mistake of assuming the changelog is just being used for the latest updates.

**Pull request merged**